src/modifierOnHitHealing.ts

```diff
@@ -0,0 +1,44 @@
+import { registerEvents, registerModifiers } from "./cards";
+import { getOrInitModifier } from "./cards/util";
+import { healUnit } from "./effects/heal";
+import * as Unit from './entity/Unit';
+import Underworld from './Underworld';
+
+// Restores (quantity) health per hit
+export const onHitHealingId = 'On Hit Healing';
+export default function registerOnHitHealing() {
+  registerModifiers(onHitHealingId, {
```

Comment:
All these new modifiers need a `cost` so we can see them in the shop.  
We can deal with the variable cost later, but for now let's make them all available for testing.

Reply:
True, I'll add this